src/documentation/pages/components/forms/select/examples/_error.vue

```diff
@@ -0,0 +1,63 @@
+<template lang="pug">
+  <ComponentExample title="Error" id="error" :tabs="exampleTabs">
+    p.-text(slot="example-description") 
+      | Use the <code>-danger</code> state to provide feedback to users when a selection has not been made. Once a selection has been made, the state must be removed. 
+      | To meet accessibility requirements, danger inputs must include an error message explaining the failure and/or how to correct it.
+    .chi-form__item(style="max-width:20rem" slot="example")
+      label(class="chi-label" for="example-er1") Label
+        abbr(class="chi-label__required" title="Required field") *
+      select(class="chi-select -danger" id="example-er1" required)
```

Comment:
As per the original documentation, this structure is only for lumen and centurylink. Portal has danger warning icon circle for Error example.
<img width="652" alt="Error" src="https://user-images.githubusercontent.com/92325462/149418161-7679e2ad-46b6-4cdf-8a48-aff8912b9b3c.png">
.

Reply:
i still see the same in both themes